ntc_templates/templates/alcatel_sros_show_service_service-using.textfsm

```diff
@@ -0,0 +1,24 @@
+Value SERVICE_ID (\d+)
+Value TYPE (\S+)
+Value ADMIN_STATUS (\S+)
+Value OPERATIONAL_STATUS (\S+)
+Value CUSTOMER_ID (\d+)
+Value SERVICE_NAME (\S+)
+
+Start
+  ^=+\s*$$
+  ^Services.*$$ -> Header
+  ^. -> Error
+
+Header
+  ^=+\s*$$
+  ^ServiceId\s+Type\s+Adm\s+Opr\s+CustomerId\s+Service\s+Name -> Services
+  ^. -> Error
+
+Services
+  ^\s*${SERVICE_ID}\s+${TYPE}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s+${CUSTOMER_ID}\s+${SERVICE_NAME}\s*$$ -> Record
+  ^-+\s*$$
+  ^Matching Services : \d+\s*$$
+  ^-+\s*$$
+  ^=+\s*$$ -> End
+  ^. -> Error
```

Comment:
Simpler.
Less duplication. More DRY.

If you like this suggestion, click the Commit suggestion button to accept my edit. Thank you!

```suggestion
Start
  ^=+\s*$$
  ^Services.*$$
  ^ServiceId\s+Type\s+Adm\s+Opr\s+CustomerId\s+Service\s+Name
  ^\s*${SERVICE_ID}\s+${TYPE}\s+${ADMIN_STATUS}\s+${OPERATIONAL_STATUS}\s+${CUSTOMER_ID}\s+${SERVICE_NAME}\s*$$ -> Record
  ^Matching\s+Services\s+:\s+\d+\s*$$
  ^-+\s*$$
  ^. -> Error
```